AlphaWallet/Transfer/Views/SendHeaderView.swift

```diff
@@ -116,49 +220,30 @@ class SendHeaderView: UIView {
-        blockchainLabel.setTitle(viewModel.blockChainName, for: .normal)
-
-        blockChainTagLabel.textAlignment = viewModel.blockChainNameTextAlignment
-        blockChainTagLabel.cornerRadius = viewModel.blockChainNameCornerRadius
-        blockChainTagLabel.backgroundColor = viewModel.blockChainNameBackgroundColor
-        blockChainTagLabel.textColor = viewModel.blockChainNameColor
-        blockChainTagLabel.font = viewModel.blockChainNameFont
-        blockChainTagLabel.text = viewModel.blockChainTag
-
-        middleBorder.backgroundColor = viewModel.borderColor
-
-        valuePercentageChangePeriodLabel.textColor = viewModel.textColor
-        valuePercentageChangePeriodLabel.font = viewModel.textLabelFont
-        valuePercentageChangePeriodLabel.text = viewModel.valuePercentageChangePeriod
-        valueChangeNameLabel.textColor = viewModel.textColor
-        valueChangeNameLabel.font = viewModel.textLabelFont
-        valueChangeNameLabel.text = viewModel.valueChangeName
-        valueNameLabel.textColor = viewModel.textColor
-        valueNameLabel.font = viewModel.textLabelFont
-        valueNameLabel.text = viewModel.valueName
+
+        placeholderLabel.textColor = viewModel.placeholderLabelTextColor
+        placeholderLabel.font = viewModel.placeholderLabelTextFont
+        placeholderLabel.text = viewModel.placeholderLabelText
+
+        marketPriceLabel.textColor = viewModel.marketPriceLabelTextColor
+        marketPriceLabel.font = viewModel.marketPriceLabelChangeFont
+        marketPriceLabel.text = viewModel.marketPriceValue
+        marketPriceLabel.isHidden = viewModel.headerViewDisplayingState.isMarketPriceHidden
+
+        tokenIconImageView.subscribable = viewModel.iconImage
+        
+        blockChainTagLabel.configure(viewModel: viewModel.blockChaintagViewModel)
+        blockChainTagLabel.isHidden = viewModel.headerViewDisplayingState.isBlockChainTagLabelHidden
+
+        changeValueContainer.isHidden = !blockChainTagLabel.isHidden
 
         valuePercentageChangeValueLabel.textColor = viewModel.valuePercentageChangeColor
-        valuePercentageChangeValueLabel.font = viewModel.textValueFont
+        valuePercentageChangeValueLabel.font = viewModel.valuePercentageChangeFont
         valuePercentageChangeValueLabel.text = viewModel.valuePercentageChangeValue
-        valueChangeLabel.textColor = viewModel.textColor
-        valueChangeLabel.font = viewModel.textValueFont
-        valueChangeLabel.text = viewModel.valueChange
-        valueLabel.textColor = viewModel.textColor
-        valueLabel.font = viewModel.textValueFont
-        valueLabel.text = viewModel.value
 
-        if viewModel.showAlternativeAmount {
-            viewsToShowOnlyIfAlternativeAmountsAreAvailable.showAll()
-        } else {
-            viewsToShowOnlyIfAlternativeAmountsAreAvailable.hideAll()
-        }
-    }
-
-    @objc private func showContractWebPage() {
-        delegate?.didPressViewContractWebPage(inHeaderView: self)
```

Comment:
Sorry, I forgot to mention this. Can you make tapping the token icon open browser at the Etherscan (or blockscout, etc) page showing this token? I think this is the line of code that does that. Before the PR you can tap on the words "Ethereum Blockchain" to trigger this.